lib/resolvers/permission.js

```diff
@@ -14,7 +14,7 @@ module.exports = ({ models }) => ({ action, data }, transaction) => {
   }
 
   if (action === 'delete') {
-    return Permission.query(transaction).delete().where('profileId', data.profileId).returning('*');
+    return Permission.query(transaction).hardDelete().where({'profileId': data.profileId, 'establishmentId': data.establishmentId}).returning('*');
```

Comment:
Can we lose the quotes around object keys where they're not required? Thanks.

Reply:
Also, the `establishmentId` condition will be required for the `update` query as well.